ui/lesson/OpRunner/OpRunner.tsx

```diff
@@ -1,9 +1,48 @@
-import React, { useEffect, useState, useRef, useCallback } from 'react'
+import React, { useEffect, useState, useRef } from 'react'
 import clsx from 'clsx'
 import LanguageExecutor from './LanguageExecutor'
 import _ from 'lodash'
 import { StatusBar } from 'ui/common'
 import { MainState, OpRunnerTypes, StackType, T } from './runnerTypes'
+import { ArcherElement } from 'react-archer'
+import { RelationType } from 'react-archer/lib/types'
+import { useArrows } from 'state/ArrowsContext'
+import useHorizontalScroll from 'hooks/useHorizontalScroll'
+
+const arrowLineStyles = {
+  startMarker: true,
+  endMarker: false,
+  strokeColor: '#3DCFEF',
```

Comment:
@benalleng @0tuedon should we perhaps align the arrow line colour with the colour of the OP code?  e.g. we could have a specific colour for OP_ADD and that would align with the arrow used for that OP code?